README.md

```diff
@@ -189,6 +196,22 @@ Translation isn't supported on AIX or Solaris, as GCC on those platforms
 doesn't support `std::locale`. In fact std::locale is buggy, so avoid
 using `get_locale` as well.
 
+To use `leatherman::local::translate` or `leatherman::locale::format`
+in your project, add these two lines to the top of your cpp file:
+
+```
+#define LEATHERMAN_i18N
```

Comment:
You don't need this if you've done `add_definitions(-DLEATHERMAN_I18N)`. Also the lower-case `i` is wrong.